qa/suites/rbd/encryption/supported-random-distro$

```diff
@@ -0,0 +1 @@
+.qa/distros/supported-random-distro$
```

Comment:
I think these are supposed to be symlinks but they are committed as files

Reply:
yep...took me ~20 minutes to figure this out...
trying to push this change via a linux VM instead of my windows host